_data/sample.yml

```
@@ -268,6 +272,22 @@ categories:
             url: https://jagat-xpub.github.io/cosmology/
           - name: *link_name_source
             url: https://github.com/jagat-xpub/cosmology
+  - name:
+      ja: 図版の多い横組書籍
+      en: Horizontal books with a lot of figures
```

```suggestion
      en: Books with many figures
```

英語では本は横組にきまってるし「Horizontal books」ではhorizontal writing modeの本のことだと分からないでしょうから「横組書籍」は単に「Books」とするのがよいでしょう。それから "Books with a lot of figures" よりも "Books with many figures" のほうがよさそうです（Google検索での件数などから）。